apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/CharSequenceExpectationsTest.kt

```diff
@@ -15,13 +16,10 @@ class CharSequenceExpectationsTest : AbstractCharSequenceExpectationsTest(
     fun1(Expect<CharSequence>::toEndWith),
     fun1(Expect<CharSequence>::notToEndWith),
     fun1(Expect<CharSequence>::toMatch),
-    fun1(Expect<CharSequence>::notToMatch)
+    fun1(Expect<CharSequence>::notToMatch),
+    property(Expect<CharSequence>::length),
+    fun1(Expect<CharSequence>::length)
 ) {
-    companion object {
```

Comment:
I see the confusion, my statement was wrong regarding we don't need the companion in api-infix. I thought there is none currently. you need to keep this. Just no need to have extra methods in the companion for `length`.